src/libstd/io/buffered.rs

```diff
@@ -364,10 +364,10 @@ impl<R: Seek> Seek for BufReader<R> {
 /// times. It also provides no advantage when writing to a destination that is
 /// in memory, like a `Vec<u8>`.
 ///
-/// When the `BufWriter` is dropped, the contents of its buffer will be written
-/// out. However, any errors that happen in the process of flushing the buffer
-/// when the writer is dropped will be ignored. Code that wishes to handle such
-/// errors must manually call [`flush`] before the writer is dropped.
+/// It is critical to call [`flush`] before `BufWriter` is dropped. Though
```

Comment:
```suggestion
/// It is critical to call [`flush`] before `BufWriter<W>` is dropped. Though
```

Reply:
Thanks for taking a look @Centril - I had excluded the the generic type to match text earlier in the description. Should I change all to include it?

```
A `BufWriter` keeps an in-memory buffer of data ...
```
and
```
`BufWriter` can improve the speed of programs that make *small* and ...
```

Reply:
Yep, please change those as well. :)